build/local/docker-compose-local.yml

```diff
@@ -170,7 +172,7 @@ services:
         ENV: dev
     volumes:
       - ./environment.json:/console/src/assets/environment.json
-    command: sh -c "ng serve --host 0.0.0.0"
+    command: sh -c "ng serve --host 0.0.0.0 --disable-host-check"
```

Comment:
ah yes @gc-ss the frontend should now also allow other domains than `localhost`

Reply:
YES! And verified!

That means the OAuth flow will still work with, say `ec2-34-207-66-192.compute-1.amazonaws.com:4200` and portforwarding is no longer required?

Reply:
Not yet fully, but it is a precondition!
We need to check weither we can change the `redirectURI` at startup or setup

Reply:
@livio-a can does know more on this topic than me

Reply:
What I am certain is that we need to register **additional** `redirectURIs`